docs/docs-content/user-management/palette-rbac/palette-rbac.md

```diff
@@ -1,376 +1,177 @@
 ---
-sidebar_label: "Palette RBAC"
-title: "Palette User Access using RBAC "
+sidebar_label: "Roles and Permissions"
+title: "Roles"
```

Comment:
Any reason you omitted Permissions in the title?

Reply:
```suggestion
title: "Roles and Permissions"
```